src/nvim/api/ui_events.in.h

```diff
@@ -0,0 +1,70 @@
+#ifndef NVIM_API_UI_EVENTS_IN_H
+#define NVIM_API_UI_EVENTS_IN_H
+
+// This file is not compiled, just parsed for definitons
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+#error "don't include this file, include nvim/ui.h"
+#endif
+
+#include "nvim/api/private/defs.h"
+#include "nvim/func_attr.h"
+#include "nvim/ui.h"
+void resize(Integer rows, Integer columns)
+    FUNC_API_SINCE(2);
+void clear(void)
+    FUNC_API_SINCE(2);
+void eol_clear(void)
+    FUNC_API_SINCE(2);
+void cursor_goto(Integer row, Integer col)
+    FUNC_API_SINCE(2);
+void mode_info_set(Boolean enabled, Array cursor_styles)
+    FUNC_API_SINCE(2);
+void update_menu(void)
+    FUNC_API_SINCE(2);
+void busy_start(void)
+    FUNC_API_SINCE(2);
+void busy_stop(void)
+    FUNC_API_SINCE(2);
+void mouse_on(void)
+    FUNC_API_SINCE(2);
+void mouse_off(void)
+    FUNC_API_SINCE(2);
+void mode_change(String mode, Integer mode_idx)
+    FUNC_API_SINCE(2);
+void set_scroll_region(Integer top, Integer bot, Integer left, Integer right)
+    FUNC_API_SINCE(2);
+void scroll(Integer count)
+    FUNC_API_SINCE(2);
+void highlight_set(HlAttrs attrs)
+  FUNC_API_SINCE(2) REMOTE_IMPL BRIDGE_IMPL;
```

Comment:
look at `flush()` and `suspend()`, these are different. But a prefix would be good.